capstone/src/main/resources/static/style.css

```diff
@@ -0,0 +1,133 @@
+body {
+  background-color: #aed1d6;
```

Comment:
wait are we making all the pages this color or will the event page be the dark color schema.

Reply:
No its gonna be the dark theme, I just haven't done any css formating yet.